src/tools/schema-tools.js

```diff
@@ -2,10 +2,9 @@
 
 const dataTools = require('./data');
 
-const makeFunction = (source, args) => {
```

Comment:
💅. I started here, but ditched the approach. Left the refactor for better readability.